packages/apps/reputation-oracle/server/src/common/config/env-schema.ts

```diff
@@ -18,7 +18,12 @@ export const envValidator = Joi.object({
   FORGOT_PASSWORD_TOKEN_EXPIRES_IN: Joi.number(),
   HCAPTCHA_SITE_KEY: Joi.string().required(),
   HCAPTCHA_SECRET: Joi.string().required(),
-  HCAPTCHA_EXCHANGE_URL: Joi.string().description('hcaptcha exchange url'),
+  HCAPTCHA_EXCHANGE_URL: Joi.string()
+    .required()
```

Comment:
You set a default value in `hcaptcha-config.service.ts` so you can remove `.required()`